editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/BoundaryForm.tsx

```diff
@@ -65,42 +150,39 @@ export default function BoundaryForm({ formikConfig, onSuccess }: FormProps) {
     },
   });
 
+  const route = useCurrentRoute();
+  const teamSlug = route.data.team;
+
+  // Cheltenham, Gloucester and Tewkesbury Strategic Local Plan
+  const slpTeams = ["cheltenham", "gloucester", "tewkesbury"];
+  const isSLPTeam = slpTeams.includes(teamSlug);
+
+  if (isSLPTeam) return <SLPInfo geojsonData={formik.values.boundaryBBox} />;
```

Comment:
Can I doublecheck a requirement here - my understanding of this one was that all SLP services/applications are exclusively going to be run from Tewkesbury "team" on PlanX - so do we actually need to apply this boundary to all three seperate teams or only Tewkesbury?

Reply:
Good catch - fixed in [030bb8c](https://github.com/theopensystemslab/planx-new/pull/4629/commits/030bb8ca5d745dd56322d98bc4f03800b666fc9e)